tooling/bundler/src/bundle/linux/rpm.rs

```diff
@@ -47,7 +47,9 @@ pub fn bundle_project(settings: &Settings) -> crate::Result<Vec<PathBuf>> {
   let license = settings.license().unwrap_or_default();
   let mut builder = rpm::PackageBuilder::new(name, version, &license, arch, summary)
     .epoch(epoch)
-    .release(release);
+    .release(release)
+    // This matches .deb compression. On a 240MB source binary the bundle will be 100KB larger than rpm's default while reducing build times by ~25%.
+    .compression(rpm::CompressionWithLevel::Gzip(6));
```

Comment:
shouldn't we make this configurable? some users will still want to shave off those 100kb

Reply:
it's not configurable for .deb either and since this was more like a hotfix (came up in a customer call) i didn't bother. We can totally look into that though. (i'm done for today)

Reply:
we have config for NSIS to control the compression, maybe we should add a similar option for linux targets as well but let's do it in another PR then